.github/actions/semver/README.md

```diff
@@ -0,0 +1,38 @@
+# Purpose
+
+This is github action with bash based script that provides semantic version generation with specific logic (different than gitversion).
```

Comment:
Please also add link to the same script we use to build docker image with semver - https://github.com/saritasa-nest/saritasa-devops-docker-images/tree/main/public/semver

That if there would be any changes in `process_dotet` and `semver` logic, `saritasa-devops-docker-images/public/semver` should be also updated

Reply:
Done